catalogs/catalogio.py

```diff
@@ -120,6 +120,18 @@ def get_control_by_id(self, control_id: str) -> dict:
         )
         return result_dict
 
+    def get_next_control_by_id(self, control_id: str):
+        search_collection = self.get_control_ids()
+        try:
+            next_idx = search_collection.index(control_id) + 1
+            return search_collection[next_idx]  # control_id was at the end of the list
```

Comment:
This comment is a good one, but I put it in the wrong spot. Should be down next to the `IndexError`. Not a blocker though

Reply:
Updated